vite.config.ts

```diff
@@ -1,7 +1,14 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
+import checker from "vite-plugin-checker";
```

Comment:
I don't see the plugin among the `devDependencies` in `package.json`, maybe you forgot to add that to the commit?

Reply:
yep I did some changes and forgot to re-add it from pnpm